fastlane/Fastfile

```diff
@@ -80,6 +80,8 @@ lane :update_hybrid_common do |options|
     "gradle/libs.versions.toml" => ["revenuecat-common = \"{x}\""],
     "core/core.podspec" => ["spec.dependency 'PurchasesHybridCommon', '{x}'"],
     "revenuecatui/revenuecatui.podspec" => ["spec.dependency 'PurchasesHybridCommonUI', '{x}'"],
+    "iosApp/Podfile" => ["  pod 'PurchasesHybridCommon', '{x}'"],
+    "iosApp/Podfile" => ["  pod 'PurchasesHybridCommonUI', '{x}'"],
```

Comment:
I think you can add them both within the same array
```suggestion
    "iosApp/Podfile" => ["  pod 'PurchasesHybridCommon', '{x}'", "  pod 'PurchasesHybridCommonUI', '{x}'"],
```

Reply:
Even better, thanks! https://github.com/RevenueCat/purchases-kmp/pull/129/commits/e8e18dde5f398d91b738e6550eb114984aff9ef1